### DISCUSSION TITLE
UI Remake

### DISCUSSION THREAD

**Discussion by geolffreym** [WatchItDev/watchit-app#173] [Ideas]
Hello community.

We are happy to share with you our app remake proposal.
Always your feedback is a key to our improvement..

Please see our figma design system:
https://www.figma.com/file/5OgbhXUQMzr4pt4jajYL7h/Watchit?type=design&node-id=0%3A1&t=ancdCk2kYqSfRfVS-1

**Comment by bennukem** [comment]
Are channel items can be an image from some provided remote metadata ?

**Reply by geolffreym** [reply]
Correct, channels are basically [orbitdb](https://github.com/orbitdb/orbit-db) addresses that contain metadata with asset information and movie details.
(Metadata is not an standard right now, that's part of the future improvement when we adopt [SEP001](https://github.com/SynapseMedia/sep/blob/main/SEP/SEP-001.md))

**Reply by geolffreym** [reply]
In a few words, each user can create their own collection and use the app to consume it (it is part of what we are also working on[nucleus](https://github.com/SynapseMedia/nucleus))
